.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,72 @@
+name: Bug report
+description: Technical issue with the Magento 2 core components
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please read [our guidelines](https://devdocs.magento.com/contributor-guide/contributing.html#report) before submitting the issue.
```

Comment:
Looks like the page was moved https://developer.adobe.com/commerce/contributor/guides/code-contributions/#report-an-issue

Reply:
good catch